packages/api-request-builder/src/create-service.js

```diff
@@ -116,13 +118,22 @@ export default function createService (
       const uri =
         (withProjectKey ? `/${options}` : '') +
         endpoint +
+        // TODO this can lead to invalid URIs as getIdOrKey can return
+        //   "/?customerId", so there can be multiple questionmarks,
```

Comment:
This was introduced in #269, I'm actually not sure why it was implemented like this 🤔